backend/src/main/scala/cromwell/backend/io/WorkflowPaths.scala

```diff
@@ -3,15 +3,15 @@ package cromwell.backend.io
 import java.nio.file.{Path, Paths}
 
 import com.typesafe.config.Config
-import cromwell.backend.{BackendJobDescriptorKey, BackendWorkflowDescriptor}
+import cromwell.backend.BackendWorkflowDescriptor
 
 object WorkflowPaths{
   val DockerRoot = Paths.get("/root")
 }
 
 class WorkflowPaths(workflowDescriptor: BackendWorkflowDescriptor, config: Config) {
   // TODO will need update when configuration is rearranged
```

Comment:
Has this TODO outlived its usefulness? I'm not sure what it could be asking for that we don't already have.